packages/number-field/test/helpers.js

```diff
@@ -0,0 +1,12 @@
+import { fire } from '@vaadin/testing-helpers';
```

Comment:
This file does not seem to be used in this PR, it probably makes sense to add it later when needed?

Reply:
Ah, it wasn't intended to be committed. Removed.